### ISSUE TITLE
[BUG] Signs doesn't render at all.

### ISSUE THREAD

**Issue by zlainsama** [MinecraftForge/MinecraftForge#2115]
I was testing my mod in latest forge, I joined couple servers, and I noticed something is wrong.
Signs doesn't render at all.

Forge Version: 1.8-11.14.3.1514
![2015-09-17_19 48 38](https://cloud.githubusercontent.com/assets/3251859/9932336/a55a5dde-5d75-11e5-9e85-d33e679a0c78.png)
![2015-09-17_19 48 41](https://cloud.githubusercontent.com/assets/3251859/9932338/a78891f2-5d75-11e5-93bd-e176de295782.png)

**Comment by jimcarreer** [comment]
If you remove all mods and only run Forge+Minecraft does the problem still occur? This is step one for blaming forge.

**Comment by zlainsama** [comment]
After restarting the minecraft(with or without the mod I was testing, I tried both), it behaves normally. I couldn't reproduce the bug.
I can't find any related error/warning in log file by myself. (http://pastebin.com/1tCudgYB)
The mod I was testing is a mod I used to test code and do some cheat stuff, this time it does not alter anything related with rendering nor adding any items/blocks/entities. (https://github.com/zlainsama/LainHelper/tree/fba23ceef8c5b3da97e6b7f5a3e378df2fd6f1b3)

I will close the issue because I couldn't reproduce the bug.
